kong.conf.default

```diff
@@ -786,6 +786,22 @@
                                         # connection may be kept open
                                         # indefinitely.
 
+#normalize_req_uri = on                 # Whether to normalize the request URI.
+                                        # If enabled, the request URI will be normalized.
+                                        #
+                                        # Examples of `normalize_req_uri = on`
+                                        # behavior:
+                                        #
+                                        # Assuming the request uri is `/test/some%20resource%20document`,
+                                        # it will be normalized as `/test/some resource document`.
+                                        #
+                                        # Valid values to this setting are:
+                                        #
+                                        # - `on`: Request URI normalization is enabled.
+                                        # - `off`: Request URI normalization is disabled.
+                                        #
+                                        # By default this value is set to `on`.
```

Comment:
normalization and decoding are two different things and have different consequences.

this description doesn't explain if the process is done only for route matching or if it affects the upstream request.

Reply:
thanks for your advice, 
now I am considering only using the unnormalized req uri to get the match_t.matches.uri_captures instead.

Then it should not affect route matching and upstream request

Reply:
Replace `normalize_req_uri` with `normalize_uri_captures` to only affect captured request URI.
Please help check again, thanks!

Reply:
this is still calling the process "normalization", but it's decoding.  i guess the function was badly named, but this change makes the error visible in the documentation and parameter name.

Reply:
Thanks for your review!
Just submit a commit to rename normalize_uri_captures to decode_uri_captures.